dcfldd: add page (#3719)
tldr-pages/tldr: pages/common/dcfldd.md

# dcfldd

> Enhanced version of dd for forensics and security.

> 

Reviewer: > More information: <http://dcfldd.sourceforge.net/>.

- Acquire a disk as raw image and hash the image using sha256 algorithm:

`dcfldd if=/dev/{{disk_device}} of={{file.img}} hash=sha256 hashlog={{file.hash}}`

- Acquire disk as raw and hash each 1GB chunk:

> `dcfldd if=/dev/{{disk_device}} of={{file.img}} hash={{hash_algorithm}} hashlog={{file.hash}} hashwindow=1G`

Reviewer 2: ```suggestion
`dcfldd if=/dev/{{disk_device}} of={{file.img}} hash={{hash_algorithm}} hashlog={{file.hash}} hashwindow={{1G}}`
```

# dcfldd

> Enhanced version of dd for forensics and security.
> More information: <http://dcfldd.sourceforge.net/>.

> - Acquire a disk as raw image and hash the image using sha256 algorithm:

Reviewer 3: Perhaps this will read better?
```suggestion
- Copy a disk to a raw image file and hash the image using SHA256:
```

`dcfldd if=/dev/{{disk_device}} of={{file.img}} hash=sha256 hashlog={{file.hash}}`

> - Acquire disk as raw and hash each 1GB chunk:

Reviewer 3: Perhaps this will read better?
```suggestion
- Copy a disk to a raw image file, hashing each 1GB chunk:
```

> `dcfldd if=/dev/{{disk_device}} of={{file.img}} hash={{hash_algorithm}} hashlog={{file.hash}} hashwindow={{1G}}`

Reviewer 4: What are the hash algorithms available ? Can we list the options directly here ?

Author: How do you suggest we list them ?

Reviewer 3: What about this? `{{sha512|sha384|sha256|sha1|hd5}}`

Author: I guess it's a good idea yeah. Do you think I should submit another pull request @sbrl  ?

Reviewer 5: @sidahmed-malaoui, I am not @sbrl but probably Sbrl will be agree with me: Yes! Of course, feel free to open a PR for this change. Thank you in advance!

Reviewer 3: Yeah, another PR would be great @sidahmed-malaoui :-)

Author: Okay guys. Pull request coming.
